tests/api/test_landing_page.py

```diff
@@ -57,7 +57,6 @@ def test_landing_page_invalid_features(example_url):
         LandingPage(**example)
 
 
-# TODO: Not sure if this actually does anything??
```

Comment:
This does the right thing, and throws an error if provided a different validation schema